vignettes/tcplfit2-vignette.Rmd

```diff
@@ -232,89 +269,95 @@ dat[wllt == "t" & acid %in% nconc[nconc > 1, acid],
+
+```{r example3_mc2, fig.height = 6, fig.width = 7}
 # If no adjustments are required for the data, the corrected value (cval) should be set as original rval
 dat[,cval := rval]
 
-## Poor well quality (wllq) wells should be removed
+# Poor well quality (wllq) wells should be removed
 dat <- dat[!wllq == 0,]
 
-## Fitting generally cannot occur if response values are NA therefore values need to be removed
+##Fitting generally cannot occur if response values are NA therefore values need to be removed
 dat <- dat[!is.na(cval),]
+```
 
-## A column for log10 concentration is added as some of the mc3 methods require logc. Given logging concentration, conc=0 are not allowed therefore a dummy non-zero value should be used
-dat[conc == 0 , conc := 0.0001]
-dat[ , logc := log10(conc)]
+### Normalization
 
-#As a final step to prepare the dataset tcplfit2 processing, a dummy aeid is required if using mc3_mthds from tcpl
-dummy_aeid <- 99999
-dat[,aeid := dummy_aeid]
+The third step normalizes and zero-centers before model fitting, and corresponds to level-3 in `tcpl`. Our example dataset has both neutral and negative controls available, and the code below demonstrates how to normalize responses to a control in this scenario. However, given experimental designs vary from assay to assay, this process also varies  across assays. Thus, the steps shown in this example may not apply to other assays and should only be considered applicable for this example data set. In other applications/scenarios, such as when neutral control or positive/negative controls are not available, the user should normalize responses that best accounts for their experimental design and data. Provided below is a list of normalizing methods used in `tcpl` for reference.
 
-## Set aeid as a key
-setkey(dat,aeid)
-```
+That being said, for this example, the normalized responses (`resp`) are calculated as a percent of control, i.e. the ratio of differences. The numerator is the difference between the corrected (`cval`) and baseline (`bval`) values and denominator is the difference between the positive/negative control (`pval`) and baseline (`bval`) values.
 
-Once the data is initialized to a point where the required fields are available, the methods included in the `tcpl` package can be identified and applied without the need for a database connection. You can see the list of available methods for Level 3 in the table below:
+$$
+\% \space control = \frac{cval - bval}{pval - bval}
+$$
 
-```{r example4_mthdlist, fig.height = 6, fig.width = 7, warning = FALSE}
-mthd_funcs <- tcpl:::mc3_mthds()
+The baseline (`bval`) values are calculated as the medians of all data in neutral control wells by plate and the negative control (`pval`) value is calculated as the plate-wise minimum, by assay plate ID (`apid`), of the medians of corrected value (`cval`) of negative control wells.
```

Comment:
This section of text needs to be clarified. Also, suggest splitting up the bval and pval descriptions into two sentences.

Reply:
Not looking at knitted markdown, but text should be more extensive now then what's in the tcplfit2 vignette currently. Thanks to @Ashley-Ko methods clean up!

Suggest not explaining each method extensively here and instead just direct to tcpl resources...